### PR TITLE
A couple of hotfixes for 2020.10.1

### LINKED ISSUE #6780: KOReader 2020.10 won't start
* KOReader version: koreader-kindle-v2020.10
* Device: Amazon Kindle 5 Classic

#### Issue
KOReader 2020.10 won't start, KOReader 2020.09 runs fine.

#### Steps to reproduce
Install KOReader 2020.10 on Kindle 5 Classic and try to run it.

##### `crash.log` (if applicable)
[crash.log](https://github.com/koreader/koreader/files/5360942/crash.log)

### PR DESCRIPTION
**Pull request by NiLuJe**
* Fix #6780
* Tweak the PB crash screen so as not to comment on unsupported behavior ;p.

---
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader/6781)

### CODE REVIEW

**Comment by NiLuJe** [comment]
(Rebase to keep commits separate).

**Comment by NiLuJe** [comment]
Ideally, I'd have marked the test as "may fail" instead, but this doesn't appear to be a busted feature?

File: platform/pocketbook/koreader.app [hunk 1]
```diff
@@ -133,8 +133,8 @@ while [ "${RETURN_VALUE}" -ne 0 ]; do
         # With a little notice at the top of the screen, on a big gray screen of death ;).
         "${KOREADER_DIR}/fbink" -q -b -c -B GRAY9 -m -y 1 "Don't Panic! (Crash n°${CRASH_COUNT} -> ${RETURN_VALUE})"
         if [ ${CRASH_COUNT} -eq 1 ]; then
-            # Warn that we're waiting on a tap to continue...
-            "${KOREADER_DIR}/fbink" -q -b -O -m -y 2 "Tap the screen to continue."
+            # Warn that we're sleeping for a bit...
+            "${KOREADER_DIR}/fbink" -q -b -O -m -y 2 "KOReader will restart in 15 sec."
```
Comment (Frenzie) [inline platform/pocketbook/koreader.app:137]: Strikes me as a bit odd compared to `15 s` or `15 seconds`.

**Review by Frenzie — APPROVED** [review]

File: platform/pocketbook/koreader.app (hunk 1 above)
Comment (NiLuJe) [reply platform/pocketbook/koreader.app:137]: I, err, couldn't remember what we'd settled on in KoboUSBMS :D.

Comment (NiLuJe) [reply platform/pocketbook/koreader.app:137]: But as it turns out, it happens to be 'sec', too ;).

**Comment by Frenzie** [comment]
Eh, whatever. ^_^

**Pull request merged**